eos-dropbox-app

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+prefix=/app
+exec /app/share/eos-dropbox-app/eos-dropbox-app.py "$@"
```

Comment:
Can this entire script just be removed if we make the eos-dropbox-app.py +x and just point the desktop file to it directly?

Reply:
Oh I guess actually it lets us set the flatpak command. I guess we can leave it, it's harmless. :)

Reply:
Actually, it WILL be removed... this morning I realized I don't need the portallauncher.py at all and so we can simplify things a lot now that neither that file nor xdg-open.py are included: stop shipping things in `share/eos-dropbox-app` and simply make `/app/bin/eos-dropbox-app` the python script.

Will include this change in the reviewed PR